app/src/test/java/org/mozilla/fenix/components/toolbar/BrowserInteractorTest.kt

```diff
@@ -18,59 +19,62 @@ import org.mozilla.fenix.quickactionsheet.QuickActionSheetController
 
 class BrowserInteractorTest {
 
+    val metrics: MetricController = mockk()
+    val context: Context = mockk()
+    val browserStore: BrowserFragmentStore = mockk(relaxed = true)
+    val browserToolbarController: BrowserToolbarController = mockk(relaxed = true)
+    val quickActionSheetController: QuickActionSheetController = mockk(relaxed = true)
+    val readerModeController: ReaderModeController = mockk(relaxed = true)
+    val session: Session = mockk()
+
+    val interactor = BrowserInteractor(
```

Comment:
I cleaned up this test quite a bit but removing all the duplicated setup code. We now have one interactor get created and used for each of the tests instead of having this code written out in every test.